constraint_utils/constraint_basics.py

```diff
@@ -821,6 +882,27 @@ def pLI(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.expr.Float32Expressi
 
 def oe_confidence_interval(ht: hl.Table, obs: hl.expr.Int32Expression, exp: hl.expr.Float32Expression,
                            prefix: str = 'oe', alpha: float = 0.05, select_only_ci_metrics: bool = True) -> hl.Table:
+    """
+    Indicate the bounds of confidence interval around the observed:expected ratio.
+    
+    For a given pair of observed and expected values, the function computes the density of the Poisson distribution
+    with fixed k (the observed number of variants) over a range of lambda values, which are given by the expected 
+    number of variants times a varying parameter ranging between 0 and 2. The cumulative density function of this 
+    function is computed and the value of the varying parameter is extracted at points corresponding to 5% and 95% 
+    to indicate the bounds of the confidence interval.
+    
+    Function will have following annotations in the output Table in addition to keys:
+        - oe_lof_lower - the lower bound of confidence interval
+        - oe_lof_upper - the upper bound of confidence interval
+
+    :param ht: Input Table with the observed and expected variant counts for LoF, missense, and synonymous variants.
+    :param obs: observed variant counts of LoF, missense, or synonymous variants.
+    :param exp: expected variant counts of LoF, missense, or synonymous variants.
```

Comment:
These are actually expressions
```suggestion
    :param obs: Expression for the observed variant counts of pLoF, missense, or synonymous variants in `ht`.
    :param exp: Expression for the expected variant counts of pLoF, missense, or synonymous variants in `ht`.
```